microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md

```diff
@@ -41,7 +41,7 @@ Command Palette provides access to Microsoft Edge commands, including various De
 
 To open Command Palette:
 
```

Comment:
Add a statement regarding the screenshot.  It currently shows old shortcut, not new Ctrl+Q shortcut.  So tagging could be:

```

The above screenshot shows the shortcut for Microsoft Edge 105-107.  Microsoft Edge 108 or later will show the shortcut `Ctrl+Q` instead.
```

Reply:
Added this line